python/sglang/srt/server_args.py

```diff
@@ -273,6 +273,10 @@ def __post_init__(self):
         ) and check_gguf_file(self.model_path):
             self.quantization = self.load_format = "gguf"
 
+        # AMD-specific Triton attention KV splits default number
+        if is_hip():
+            self.triton_attention_num_kv_splits = 16
+
```

Comment:
@whchung we noticed that default 8 works better to most cases? Is 16 slightly better as you see, or better toward long sequences as you interested?

Reply:
@this PR boost grok, etc. as observed.